Proposta Guidelines Allontanamento Ambassador (#25)
Il-Libro-Open-Source/governance: guidelines/ambassador-allontanamento.md

Quando viene usato il termine _sondaggio standard infra-gruppo_ si intende un sondaggio con quorum al 50% e maggioranza semplice della durata di 48 ore tra i membri del gruppo di riferimento.

Il mancato raggiungimento del quorum in un sondaggio standard infra-gruppo corrisponderà all'apertura del medesimo sondaggio, con le medesime condizioni, rivolto a tutti i membri del team Ambassador.

Un membro del team Ambassador può essere allontanato per i seguenti motivi:

- Mancato rispetto del [Codice di Comportamento](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_ nel Governance Group.
- Violazione delle linee guida imposte dal Communications Group all'interno del gruppo Telegram. In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
- Violazione delle linee guida imposte dal Relations Group all'interno del perimetro di azione di quest'ultimo (Conferenze, post pubblici, ecc.). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.

## Indice di gradimento

Ogni trimestre verrà effettuato un sondaggio contenente l'intera lista dei membri del team Ambassador.  

> Il sondaggio sarà anonimo, a risposta multipla e avrà una durata di 15 giorni.  

Author: ```suggestion
Il sondaggio sarà a risposta multipla e avrà una durata di 15 giorni. I risultati del sondaggio verranno visualizzati - e condivisi in maniera anonimizzata - dal Governance Group in un apposito meeting nel primo momento utile passati i giorni di votazione.
```

> Qualora un membro del team Ambassador non raggiunga il 50% di gradimento, verrà allontanato dal team.

Author: ```suggestion
Qualora un membro del team Ambassador non raggiunga il 50% di gradimento, verrà fatta una valutazione da parte del Governance Group che, a maggioranza semplice, deciderà se allontanare o meno la persona dal team Ambassador.
```

> La mancanza di votazione in questo sondaggio senza giustificati motivi verrà considerata come mancanza di partecipazione e verrà aperto un _sondaggio standard infra-gruppo_ nel Governance Group.

Author: ```suggestion
La mancanza di votazione in questo sondaggio senza giustificati motivi verrà considerata come mancanza di partecipazione e la figura subirà la medesima valutazione di cui sopra da parte del Governance Group.
```

La motivazione di questa regola è quella di garantire un team sempre attivo e partecipe, che riduca al minimo gli attriti e che possa valutare l'impatto di ogni suo membro ed esprimere il proprio giudizio a prescindere dalle linee guida che possono essere facilmente aggirate.

## Procedura di re-inserimento

> L'allontanamento non consentirà una nuova candidatura classica e potrà essere revocato solo in seguito ad un nuovo sondaggio con quorum al 50% e maggioranza pari al 75% dei votanti nel team Ambassador.

Author: ```suggestion
```

> Deve trascorrere almeno un mese di calendario tra la data di allontanamento e la data in cui viene presentata la richiesta di re-inserimento.

Author: ```suggestion
Devono trascorrere almeno tre mesi di calendario tra la data di allontanamento e la data in cui viene presentata la richiesta di re-inserimento.
```

Un membro del team Ambassador può essere allontanato per i seguenti motivi:

- Mancato rispetto del [Codice di Comportamento](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_ nel Governance Group.
- Violazione delle linee guida imposte dal Communications Group all'interno del gruppo Telegram. In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
- Violazione delle linee guida imposte dal Relations Group all'interno del perimetro di azione di quest'ultimo (Conferenze, post pubblici, ecc.). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
- Non partecipazione ad almeno il 50% dei sondaggi proposti in un periodo di 30 giorni senza giustificati motivi. In tal caso, sarà il gruppo stesso a decidere se allontanare o meno la figura con un sondaggio tra i propri membri con quorum al 50% e maggioranza semplice della durata di 48 ore.
- Mancata assegnazione di un capitolo entro 30 giorni dalla data di nomina. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per assegnarsi un capitolo.

> - Mancata stesura di contenuti propri o review di contenuti altrui nei capitoli assegnati entro 60 giorni dall'ultima stesura e/o review. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per dare una motivazione plausibile. A risposta ricevuta, verrà richiesto un _sondaggio standard infra-gruppo_ per la sua rimozione. Questa regola non si applica ai membri del team Ambassador assegnati a capitoli che non hanno ricevuto alcuna proposta di modifica.

Reviewer: Non sarebbe meglio introdurre una sorta di "cartellino giallo" (in determinate occasioni, ovvio) per evitare che qualcuno venga espulso così dal nulla?
Servirebbe per evitare che qualcuno violi uno dei punti e subito si becca il rischio espulsione (basterebbe che qualcuno fosse aggiornato con le linee guida redatte dai Communications e dai Relations).

Author: L'ho indicata come one-shot perché sta alla sensibilità di ognuno decidere se si sente offeso da quella violazione tanto da segnalarla.. Che dici, meglio specificarlo?

Reviewer: Nah, alla fine se ne può benissimo discutere prima di arrivare alla segnalazione.

Author: Se ti ho convinto mettila resolved così sappiamo non essere un punto aperto

Reviewer 2: Con infra-gruppo si intende il perimetro ristretto del focus group della persona? O di tutto il gruppo degli Ambassador? Mi sembra sia più logica la seconda.

Altro tema, da capire in caso come gestire il sondaggio tra tutti gli Ambassador - 1 (la persona nominata per espulsione), ovviamente non Telegram.

Inoltre, da capire come comunicare il fatto che si è in corsa per una possibile espulsione e valutare come gestire eventuali reazioni. Cioè: se ti stiamo dicendo che c'è una votazione per espellerti e poi resti, con che morale rimani? Secondo me qui la questione è ancora più spinosa.

Ragionandoci, io **eviterei il sondaggio**. Punterei direttamente sulla chiarezza delle regole e sulla logica dei cartellini come suggerito da Corrado. Se non rispetti le regole, vieni prima avvisato e poi espulso, non dovrebbe esserci un "sondaggio post-regole"

Author: > Con infra-gruppo si intende il perimetro ristretto del focus group della persona? O di tutto il gruppo degli Ambassador? Mi sembra sia più logica la seconda.

No, intendo nel working group di riferimento. Se l'utente non rispetta il codice di comportamento, è il Governance a decidere se espellerlo. Se non rispetta le regole di comportamento tra Ambassador del Communications, è il Communications a decidere se espellerlo. Sono i WG di riferimento i responsabili della stesura delle proprie regole, è giusto che lo siano anche nell'applicazione.

> Altro tema, da capire in caso come gestire il sondaggio tra tutti gli Ambassador - 1 (la persona nominata per espulsione), ovviamente non Telegram.

Ovviamente, questo è un tema da trattare dopo aver deciso le linee guida.

> Inoltre, da capire come comunicare il fatto che si è in corsa per una possibile espulsione e valutare come gestire eventuali reazioni. Cioè: se ti stiamo dicendo che c'è una votazione per espellerti e poi resti, con che morale rimani? Secondo me qui la questione è ancora più spinosa. Ragionandoci, io eviterei il sondaggio. Punterei direttamente sulla chiarezza delle regole e sulla logica dei cartellini come suggerito da Corrado. Se non rispetti le regole, vieni prima avvisato e poi espulso, non dovrebbe esserci un "sondaggio post-regole"

Lascio il parere anche ad altri, continuo a credere che il sondaggio sia la cosa migliore, altrimenti ci toccherebbe burocratizzare troppo la cosa.. Se scriviamo tra le regole "Non bisogna bestemmiare" uno potrebbe risponderti che "Dalle sue parti quella parola non è una bestemmia", invece così facciamo in modo che chiunque possa valutare ogni singolo caso nel giusto contesto in cui è calato.

Author: @Il-Libro-Open-Source/ambassadors

## Indice di gradimento

Ogni trimestre verrà effettuato un sondaggio contenente l'intera lista dei membri del team Ambassador.  
Il sondaggio sarà anonimo, a risposta multipla e avrà una durata di 15 giorni.  
Qualora un membro del team Ambassador non raggiunga il 50% di gradimento, verrà allontanato dal team.
La mancanza di votazione in questo sondaggio senza giustificati motivi verrà considerata come mancanza di partecipazione e verrà aperto un _sondaggio standard infra-gruppo_ nel Governance Group.

La motivazione di questa regola è quella di garantire un team sempre attivo e partecipe, che riduca al minimo gli attriti e che possa valutare l'impatto di ogni suo membro ed esprimere il proprio giudizio a prescindere dalle linee guida che possono essere facilmente aggirate.

## Procedura di re-inserimento

> L'allontanamento non consentirà una nuova candidatura classica e potrà essere revocato solo in seguito ad un nuovo sondaggio con quorum al 50% e maggioranza pari al 75% dei votanti nel team Ambassador.

Reviewer: ```suggestion
L'allontanamento non consentirà una nuova candidatura classica e potrà essere revocato solo in seguito ad un nuovo sondaggio con quorum al 50% e maggioranza pari al 75% dei votanti nel team Ambassador.
Deve trascorrere almeno un mese di calendario tra la data di allontanamento e la data in cui viene presentata la richiesta di re-inserimento.
```

Il mancato raggiungimento del quorum in un sondaggio standard infra-gruppo corrisponderà all'apertura del medesimo sondaggio, con le medesime condizioni, rivolto a tutti i membri del team Ambassador.

Un membro del team Ambassador può essere allontanato per i seguenti motivi:

- Mancato rispetto del [Codice di Comportamento](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_ nel Governance Group.
- Violazione delle linee guida imposte dal Communications Group all'interno del gruppo Telegram. In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
- Violazione delle linee guida imposte dal Relations Group all'interno del perimetro di azione di quest'ultimo (Conferenze, post pubblici, ecc.). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
- Non partecipazione ad almeno il 50% dei sondaggi proposti in un periodo di 30 giorni senza giustificati motivi. In tal caso, sarà il gruppo stesso a decidere se allontanare o meno la figura con un sondaggio tra i propri membri con quorum al 50% e maggioranza semplice della durata di 48 ore.

> - Mancata assegnazione di un capitolo entro 30 giorni dalla data di nomina. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per assegnarsi un capitolo.

Reviewer 3: Su questo punto non concordo, per me un Ambassador può non prendere in assegnazione un topic specifico, ma può essere attivo nella scrittura di alcuni senza esserne OWNER! 

Nella mia visione l'ambassador è un contributor molto attivo nel progetto e con attivo intendo che scrive cose, commenta issue e fa review!

Questo rende molto aperto il fatto di essere Ambassador che è ottimo per una community Open, poi solo se vuole ulteriori incarichi può diventare OWNER di un capitolo (andrebbe definito come diventare OWNER) o entrando in un wg!

Author: Io vedo l'owner di un capitolo come persona responsabile dello stesso, che sia o meno da sola nel farlo, per quello indicavo la necessità..
Possiamo anche metterci in 7 sul testing e contribuire tutti assieme, ma ognuno dei 7 sarà assegnatario dell'argomento.

Author: Quindi tu vedi due tipi di Ambassador su un argomento, uno che faccia da owner e uno che faccia da contributor? Non è troppo complicata?

Reviewer 3: In realtà è più semplice... Ambassador è uno che contribuisce in modo continuativo, poi se e solo se la sente si prende responsabilità aggiuntive diventando OWNER di un capitolo o entrando in un WG.

Io mi immagino una cosa molto semplice, una persona che si diverte a contribuire in maniera utile al progetto, ma non se la sente di avere ruoli più complessi per tempo o più semplicemente perché non se la sente, però mi pare giusto che gli venga riconosciuto un lavoro molto spinto sul progetto.

Lo vedo come modo di rendere più inclusivo il tutto...

Esempio: contribuisci spesso a Kubernetes entri nella Org e puoi fare il reviewer e tutto, ma non sei obbligato a prenderti leadership di un progetto (per noi capitolo)

Author: Però così rischiamo il problema opposto.. Da 40 Ambassador diventiamo 500 :)

Reviewer 2: Concordo, capisco il voler essere inclusivi e mi piace come approccio, ma non possiamo neppure gestire un numero di Ambassador troppo elevato.

@davideimola mi sembra che tu abbia espresso il concetto di "contributor molto attivo" - se non sei owner di un capitolo o non partecipi in un WG mi sembra difficile attribuire il ruolo di Ambassador, poi però ci eravamo anche proposti di inserire un ruolo di "amico del progetto", ricordi? Potrebbe essere ciò a cui ti riferisci?

> - Non partecipazione ad almeno il 50% dei sondaggi proposti in un periodo di 30 giorni senza giustificati motivi. In tal caso, sarà il gruppo stesso a decidere se allontanare o meno la figura con un sondaggio tra i propri membri con quorum al 50% e maggioranza semplice della durata di 48 ore.
> - Mancata assegnazione di un capitolo entro 30 giorni dalla data di nomina. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per assegnarsi un capitolo.
> - Mancata stesura di contenuti propri o review di contenuti altrui nei capitoli assegnati entro 60 giorni dall'ultima stesura e/o review. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per dare una motivazione plausibile. A risposta ricevuta, verrà richiesto un _sondaggio standard infra-gruppo_ per la sua rimozione. Questa regola non si applica ai membri del team Ambassador assegnati a capitoli che non hanno ricevuto alcuna proposta di modifica.

Author: ```suggestion
```

## Indice di gradimento

Ogni trimestre verrà effettuato un sondaggio contenente l'intera lista dei membri del team Ambassador.  
Il sondaggio sarà anonimo, a risposta multipla e avrà una durata di 15 giorni.  
Qualora un membro del team Ambassador non raggiunga il 50% di gradimento, verrà allontanato dal team.
La mancanza di votazione in questo sondaggio senza giustificati motivi verrà considerata come mancanza di partecipazione e verrà aperto un _sondaggio standard infra-gruppo_ nel Governance Group.

> La motivazione di questa regola è quella di garantire un team sempre attivo e partecipe, che riduca al minimo gli attriti e che possa valutare l'impatto di ogni suo membro ed esprimere il proprio giudizio a prescindere dalle linee guida che possono essere facilmente aggirate.

Reviewer 3: Questa motivazione non mi fa impazzire ad essere sincero... potrebbe essere usato nel modo sbagliato. Uno può essere antipatico o di carattere forte, ma se contribuisce e fa il suo non mi sembra corretto nei confronti della persona allontanarlo solo perchè in modo anonimo sta antipatico a qualcuno...

Alla fine se non viola niente del regolamento o del CoC è gisto che resti!

Aggiungo che potrebbe creare secondo me una community che decide per antipatie o simpatie e creando quindi una "casta", passatemi il termine.

Author: Questo rimane un po' un nodo da capire bene, nel senso che in generale sono d'accordo con te, ma non vorrei trovarmi nella situazione per cui "Siamo in 50, a tutti noi non sta bene il modo di comportarsi di 1 di noi, ma non violando il CoC rimane", anche se vista l'attuale formazione dubito sia una situazione che accadrà 🚀

Reviewer 3: Capisco il punto di vista, però forse implicitamente violi la CoC, comunque probabilmente si potrebbe anche pensare che si possano mandare delle segnalazioni e il wg cercherà di sistemare questi attriti nel caso prima che sfocino e peggiorino!

Author: A questo punto ha senso che il Governance Group abbia potere di emettere un sondaggio di "sfiducia" in qualche modo.
